api-spec/api-spec.md

```diff
@@ -104,3 +156,57 @@ All Extensions **should** use attribute names qualified from the root of Item, r
 | query     | string \| QueryFilter | Placeholder parameter for [API Query Extension](extensions/query/README.md) query value. |
 
  **query** Represents a query in the query language.
+
+### Paging Extension
+
+OGC API supports paging through hypermedia links and STAC follows the same pattern for the cross collection search. For 
+GET requests, a link with `rel` type `next` is supplied.  This link may contain any URL parameter that is necessary 
+for the implementation to understand how to provide the next page of results, eg: `page`, `next`, `token`, etc.  The 
+parameter name is defined by the implementor and is not necessarily part of the API specification.  For example:
+
+```json
+{
```

Comment:
I guess it would be good to also have a more complex example with method, headers etc, too.

Reply:
I cleaned up the examples and added an additional example using headers.